mongoCluster/Driver.cs

```diff
@@ -499,12 +660,35 @@ private bool _queryJoin(String firstCollection, String secondCollection)
         /// </summary>
         /// <param name="collectionName">String representing the collection</param>
         private bool _queryUpdate(String collectionName)
-        { 
+        {
             /*  Implementation Strategy:
             *      1. Perform an updateMany on all listings from Portland 
             *         with greater than 2 bedrooms and 2 bathrooms to set the require_guest_phone_verification field as true. 
             */
             // TODO: stub
+
```

Comment:
Would it be better to make a quick function for this instead of adding it manually every time we need it?